package.json

```diff
@@ -30,9 +30,9 @@
     "eslint-plugin-prettier": "^3.3.0",
     "lerna": "^5.6.2",
     "prettier": "^1.14.3",
-    "typedoc": "^0.22.5",
+    "typedoc": "^0.23.28",
     "typedoc-plugin-markdown": "^3.11.3",
-    "typescript": "~4.3.5"
+    "typescript": "~4.7.4"
```

Comment:
Few points here:
* _typescript_ version should be the same like we are using in base packages (~4.9.3)
* latest _typedoc_ version is 0.24.6, should we just use latest?

Reply:
I tried updating them to the latest version but then `sitecore-jss-angular-schematics` package failed for doc generation.
I tried updating typescript in `sitecore-jss-angular-schematics` but that lead to another error. This was the only stable combination until now and we are using typescript 4.7 in most of the packages.

Reply:
As a workaround it's fine for now